extension/lib/LogHandler.jsm

```diff
@@ -51,8 +51,10 @@ this.LogHandler = {
   },
 
   shutdown() {
-    clearTimeout(intervalId);
-    timerManager.unregisterTimer(TIMER_NAME);
+    if (intervalId) {
+      clearTimeout(intervalId);
+      timerManager.unregisterTimer(TIMER_NAME);
+    }
```

Comment:
This isn't enough, because you still import `LogManager` for the first time in `Bootstrap.shutdown()`, causing `Failed to load module resource://pioneer-study-pathfinder/lib/LogHandler.jsm.` as you (I think) can't import things during `Bootstrap.shutdown()`.